services/src/main/java/org/keycloak/protocol/oidc/grants/ciba/endpoints/BackchannelAuthenticationCallbackEndpoint.java

```diff
@@ -56,6 +56,7 @@ public class BackchannelAuthenticationCallbackEndpoint extends AbstractCibaEndpo
     private static final Logger logger = Logger.getLogger(BackchannelAuthenticationCallbackEndpoint.class);
 
     private final HttpRequest httpRequest;
+    protected AuthenticationChannelResponse authenticationChannelResponse;
```

Comment:
@patrick-primesign - while I leave the final comment for the core team, here is something I noticed: 

You make this protected so a derived class can access it. At the same time you're passing it in to some methods. Can we be consistent to only pass it to methods, as I would assume it would make the code simpler to follow?

As you've been adding protected methods / making methods protected, it might be good to add some Javadoc to them explaining how they are intended to be used.

Reply:
@ahus1 yeah, I agree, we needed it in our implementation for getRawBearerToken(), which normally would use the http headers. 
I will add response and headers as parameters getRawBearerToken, to make it more readable or consistent.
Javadoc I will also add.

Reply:
I pushed some changes to the commit.